src/picongpu/include/simulation_defines/param/ionizer.param

```diff
@@ -19,7 +19,28 @@
 
 /* @file ionizer.param
  *
- * Ionization Model Configuration
+ * This file contains the proton and neutron numbers of commonly used elements
+ * of the periodic table. The elements here should have a matching list of
+ * ionization energies in @see ionizationEnergies.param. Moreover this file
+ * contains a description of how to configure an ionization model for a species.
+ * Currently each species can only be assigned exactly one ionization model.
```

Comment:
> Currently each species can only be assigned exactly one ionization model.

This does not apply anymore. :)

`/* Each species can be assigned a list of ionization models that will be applied sequentially */`